tmail-backend/apps/distributed/src/doc/release-instructions/1.0.17/index.md

```
@@ -0,0 +1,99 @@
+```
+
+Adapt the signatures to your needs.
+
+### Tiering on thread table (Optional)
+
+Depending on the environment you might need to enable some tiering option to save data on some Cassandra tables.
+One of those is tiering on the `thread_2` Cassandra table.
+
+If you need to turn it on, add in `jvm.properties`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)